source/jormungandr/jormungandr/interfaces/v1/Calendars.py

```diff
@@ -83,42 +86,45 @@
 
 class Calendars(ResourceUri):
     def __init__(self):
-        ResourceUri.__init__(self)
-        self.parsers = {}
-        self.parsers["get"] = reqparse.RequestParser(
-            argument_class=ArgumentDoc)
+        ResourceUri.__init__(self, output_type_serializer=api.CalendarsSerializer)
         parser_get = self.parsers["get"]
-        parser_get.add_argument("depth", type=int, default=1)
+        parser_get.add_argument("depth", type=depth_argument, default=1,
+                                help="The depth of your object")
         parser_get.add_argument("count", type=default_count_arg_type, default=10,
                                 help="Number of calendars per page")
         parser_get.add_argument("start_page", type=int, default=0,
                                 help="The current page")
         parser_get.add_argument("start_date", type=six.text_type, default="",
-                                help="Start date")
+                                help="The current page")
         parser_get.add_argument("end_date", type=six.text_type, default="",
                                 help="End date")
-        parser_get.add_argument("forbidden_id[]", type=six.text_type,
+        parser_get.add_argument("forbidden_id[]", type=six.text_type, deprecated=True,
                                 help="DEPRECATED, replaced by `forbidden_uris[]`",
                                 dest="__temporary_forbidden_id[]",
                                 default=[],
-                                action='append')
+                                action='append',
+                                schema_metadata={'format': 'pt-object'})
         parser_get.add_argument("forbidden_uris[]", type=six.text_type,
                                 help="forbidden uris",
                                 dest="forbidden_uris[]",
                                 default=[],
-                                action='append')
+                                action="append",
+                                schema_metadata={'format': 'pt-object'})
         parser_get.add_argument("distance", type=int, default=200,
                                 help="Distance range of the query. Used only if a coord is in the query")
+        parser_get.add_argument("_current_datetime", type=DateTimeFormat(),
+                                schema_metadata={'default': 'now'}, hidden=True,
+                                default=datetime.utcnow(),
+                                help="The datetime we want to publish the disruptions from."
```

Comment:
disruptions? (if it's wrong we may have problems elsewhere as there are a lot of copy-paste on that I guess)

Reply:
I don't get it, that's normal that disruptions is in plural form.